### PR TITLE
[CoreLocationUI] Add new framework for Xcode13 beta1.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: tools/common/Frameworks.cs [hunk 1]
```diff
@@ -484,6 +486,8 @@ public static Frameworks GetwatchOSFrameworks (bool is_simulator_build)
 
 				{ "Accessibility", "Accessibility", 7,0 },
 				{ "UniformTypeIdentifiers", "UniformTypeIdentifiers", 7,0 },
+
+				{ "CoreLocationUI", "CoreLocationUI", 8,0 },
```
Comment (spouliot) [inline tools/common/Frameworks.cs:490]: why ? it does not look like it's available for watchOS

File: src/corelocationui.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	public enum CLLocationButtonLabel : long {
+		None = 0,
+		CurrentLocation,
+		SendCurrentLocation,
+		SendMyCurrentLocation,
+		ShareCurrentLocation,
+		ShareMyCurrentLocation,
+	}
+
+	[NoTV, NoMac, NoWatch, iOS (15,0), MacCatalyst (15,0)]
```
Comment (spouliot) [inline src/corelocationui.cs:29]: `API_AVAILABLE(ios(15.0),watchos(8.0))`
I guess it's not in the watchOS header ?

**Review by spouliot — CHANGES_REQUESTED** [review]
watchOS is inconsistent 
if the API were not added (like headers suggest) then it should not be defined for watchOS frameworks

**Review by rolfbjarne — APPROVED** [review]

File: tools/common/Frameworks.cs (hunk 1 above)
Comment (mandel-macaque) [reply tools/common/Frameworks.cs:490]: I think we can remove it and add the ignore

**Review by spouliot — APPROVED** [review]

File: tools/mtouch/Makefile [hunk 3]
```diff
@@ -169,6 +169,8 @@ SIMLAUNCHER_FRAMEWORKS =  \
 	-weak_framework NearbyInteraction \
 						\
 	-weak_framework AdServices	\
+						\
+	-weak_framework CoreLocationUI \
```
Comment (tj-devel709) [inline tools/mtouch/Makefile:173]: What is this for?

File: tools/common/Frameworks.cs [hunk 4]
```diff
@@ -418,6 +418,8 @@ public static Frameworks CreateiOSFrameworks (bool is_simulator_build)
 
 				{ "AdServices", "AdServices", 14,3 },
 
+				{ "CoreLocationUI", "CoreLocationUI", 15,0 },
```
Comment (tj-devel709) [inline tools/common/Frameworks.cs:421]: When do we do this?

Comment (mandel-macaque) [reply tools/common/Frameworks.cs:421]: When a new framework is added.

**Comment by mandel-macaque** [comment]
Known issues: https://github.com/xamarin/maccore/issues/2443

**Pull request merged**